wolvic/browser/vr/wvr_manager.cc

```diff
@@ -605,6 +604,7 @@ void WvrManager::WebXrTryStartAnimatingFrame() {
     return;
   }
 
+  base::TimeTicks now = base::TimeTicks::Now();
```

Comment:
We may not use the variable if we early return in line 613, so what about moving the variable declaration and assignment closer to where it's used ? 

As a matter of fact, is the variable used anywhere else ? Have you considered calling  base::TimeTicks::Now() directly when computing the time_delta ?

Reply:
I don't think the spec mentions anything regarding the precise time to fetch the time delta. That said, I have noticed that the other implementations (GVR, cardboard and Arcore) get that just before fetching a fresh pose, so that's why I'm a bit reluctant on calling `::Now()` directly when computing the time_delta.

I have nevertheless uploaded a new commit that brings `::Now()` after the return call. PTAL and thank you for the nice observations, Javi!